predicators/envs/stick_button.py

```diff
@@ -416,7 +416,11 @@ def _Pressed_holds(state: State, objects: Sequence[Object]) -> bool:
     @classmethod
     def Above_holds(cls, state: State, objects: Sequence[Object]) -> bool:
         """Public for use by oracle options."""
-        obj1, obj2 = objects
+        if len(objects) == 3:
```

Comment:
what is the meaning of Above(x, y, z)?

Reply:
woops - this was a vestigial hack; my b!